packages/neuron-wallet/tests/block-sync-renderer/indexer-cache-service.intg.test.ts

```diff
@@ -88,7 +103,7 @@ const mockGetTransactionHashes = (mocks: any[] = []) => {
 
   for (const lock of [formattedDefaultLockScript, formattedAcpLockScript, formattedLegacyAcpLockScript]) {
     const { hashes } = mocks.find(mock => mock.lock === lock) || { hashes: [] }
-    stubbedConstructor.calledWith(expect.anything(), { lock }).mockReturnValue({
+    stubbedConstructor.calledWith(expect.anything(), { lock }, ckbRpcUrl, { includeStatus: false }).mockReturnValue({
```

Comment:
I see that the type definition of `TransactionCollector` is as follows. Are these four parameters reasonable?
```
declare class TransactionCollector {
  constructor(
    indexer: Indexer,
    queries: QueryOptions,
    options?: TransactionCollectorOptions
  );
}
```

Reply:
It's from the '@nervina-labs/ckb-indexer'.  Maybe you find the base definition?

<img width="600" alt="image" src="https://github.com/nervosnetwork/neuron/assets/22511289/d04fa85a-0cfd-481f-92ee-3ed27006de51">
<img width="600" alt="image" src="https://github.com/nervosnetwork/neuron/assets/22511289/5e1645e5-5e45-4ce9-b162-7c7c29b6983d">

Reply:
Oh, indeed.